run/run_test.go

```diff
@@ -28,6 +29,18 @@ func TestRun(t *testing.T) {
 		t.Fatalf("failed to initialize a testing option: %v", err)
 	}
 
+	var jourctlOpt *option.Option
+	var jerr error
+	if *subject == "finch" {
+		jourctlOpt, jerr = option.New(
+			[]string{"/Applications/Finch/lima/bin/limactl", "shell", "finch", "sudo", "journalctl"},
```

Comment:
This path is only available when using the installer right? It won't work if building from source

Reply:
This does seem a bit hacky, but making this configurable and passing through Makefile might be also too complicated. I feel we can live with this, and if we need to test the finch that's built from source, we may run `make test-e2e` in the finch repo with (possibly) temporarily replacing the `common-test` version with the local one in CLI's `go.mod`.

Reply:
Do you think it's okay to only check if the subject string equals "finch" as it may contains the path to to `finch` executable?

Reply:
I feel that the current form is probably fine. Could you elaborate more (e.g., maybe some code) about the proposed change?

Reply:
This still won't work if we're building from source. We'll still be invoking `journalctl` in the VM installed by the installer.